src/Microsoft.AspNetCore.Mvc.Razor/Internal/RazorPagePropertyActivator.cs

```diff
@@ -27,11 +27,21 @@ public class RazorPagePropertyActivator
             Type modelType,
             IModelMetadataProvider metadataProvider,
             PropertyValueAccessors propertyValueAccessors)
+            : this(pageType, modelType, null, metadataProvider, propertyValueAccessors)
+        {
+        }
+
+        public RazorPagePropertyActivator(
```

Comment:
This is just an additional constructor to avoid a breaking change. This class is just manually instantiated

Reply:
This is `.Internal` so it's OK to just modify the existing constructor.

Reply:
@javiercn - what's your plan for this?

Reply:
Discussed in person, @javiercn is going to remove the old constructor.